.changeset/gentle-seahorses-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now see now changes in URL when click "Select channel" in product list
```

Comment:
Suggestion: I would change the changeset a bit

Maybe `"Select channel" button in product list no longer changes the URL` ?